napari/_vispy/visuals/points.py

```diff
@@ -25,5 +25,14 @@ def symbol(self):
 
     @symbol.setter
     def symbol(self, value):
-        for subv in self._subvisuals[:2]:
-            subv.symbol = value
+        for marker in self._subvisuals[:2]:
+            marker.symbol = value
+
+    @property
+    def scaling(self):
+        return self._subvisuals[0].scaling
+
+    @scaling.setter
+    def scaling(self, value):
+        for marker in self._subvisuals[:2]:
+            marker.scaling = value
```

Comment:
I think it would be nice to add a short docstring explaining the scaling property.